src/components/AdminDashboard.jsx

```diff
@@ -73,7 +71,7 @@ class AdminDashboard extends React.Component {
       <div>
         <TopNav title={title} backToURL={backToURL} />
         <div className={css(styles.container)}>
-          {this.renderNavigation(sections)}
+          {this.renderNavigation()}
```

Comment:
We were previously passing `sections` to `renderNavigation()`, but never using it within that function. ESLint flagged the unused variable, so to resolve that, I removed the parameter from both the function definition and the call.